src/main/java/uk/gov/hmcts/befta/util/MapPlaceholders.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.befta.util;
+
+public enum  MapPlaceholders {
```

Comment:
The name is a bit obscure.. How about, ExpectedValuePlaceholder? (Singular, not plural.)

Reply:
Changed the name